ui/redux/reducers/app.js

```diff
@@ -45,12 +45,14 @@ const defaultState: AppState = {
   modal: null,
   modalProps: {},
   platform: process.platform,
-  upgradeSkipped: sessionStorage.getItem('upgradeSkipped') === 'true',
   daemonVersionMatched: null,
   daemonReady: false,
   hasSignature: false,
   badgeNumber: 0,
+  // @if TARGET='app'
+  upgradeSkipped: sessionStorage.getItem('upgradeSkipped') === 'true',
   volume: Number(sessionStorage.getItem('volume')) || 1,
```

Comment:
volume should still be included for lbry.tv. You could also move it above the reducers and use a try catch.